ChangeLog.zephyr.md

```diff
@@ -0,0 +1,8 @@
+# The List of changes to fit zephyr.
+
+Need to take care to not break these changes when updating pico-sdk.
+
+## [cd0bef7](https://github.com/zephyrproject-rtos/hal_rpi_pico/commit/cd0bef7) - 2022-08-26
+ - Rename adc_read() to pico_adc_read()
+## [191f5ba](https://github.com/zephyrproject-rtos/hal_rpi_pico/commit/191f5ba) - 2022-02-02
+ - Patch occurrences of KHZ/MHZ to PICO_KHZ/PICO_MHZ
```

Comment:
The commit SHAs given here are from the PR branch and will no longer be valid once this branch is merged.

I would suggest adopting the approach documenting the list of patches and the files changed: https://github.com/zephyrproject-rtos/hal_st/blob/cccbc24c14decfd3f93959f7b14514536af973c7/sensor/stmemsc/README#L60-L62

If one wants to find out which commit, they can easily do so by using git blame anyway.

Reply:
Yep, please do the suggested change @soburi